package.json

```diff
@@ -19,6 +19,7 @@
     "debug": "^2.2.0",
     "drag-drop": "^2.11.0",
     "electron-localshortcut": "^0.6.0",
+    "electron-prebuilt": "0.37.2",
```

Comment:
I will double check that making this change does not cause the packaged builds to start including `electron-prebuilt` in the `node_modules` folder that we ship to users.